packages/web/src/components/gcds-button/gcds-button.css

```diff
@@ -86,38 +86,6 @@
     }
   }
 
-  &.button--text-only {
-    background-color: var(--gcds-button-text-only-default-background);
-    color: var(--gcds-button-text-only-default-text);
-
-    &:not([aria-disabled='true']) {
-      text-decoration: underline;
-      text-underline-offset: 0.2em;
-      text-decoration-color: currentColor;
-      text-decoration-thickness: var(--gcds-button-text-only-default-decoration-thickness);
-    }
-
-    &:hover {
-      background-color: var(--gcds-button-text-only-default-background);
-      color: var(--gcds-button-text-only-hover-text);
-      text-decoration-thickness: var(--gcds-button-text-only-hover-decoration-thickness);
-    }
-
-    &:focus {
-      background-color: var(--gcds-button-shared-focus-background);
-      color: var(--gcds-button-shared-focus-text);
-      outline: var(--gcds-button-shared-focus-outline-width) solid var(--gcds-button-shared-focus-background);
-      outline-offset: var(--gcds-button-border-width);
-      border-color: var(--gcds-button-shared-focus-background);
-      text-decoration: none;
-    }
-
-    &:active {
-      background-color: var(--gcds-button-shared-active-background);
-      border-color: var(--gcds-button-shared-active-background);
-    }
-  }
-
```

Comment:
I can't comment on the right line cuz Github (:D) but for lines 169 + 170:
Since we won't be supporting the `text-only` style for buttons anymore, can we update these 2 lines to remove the complexity that is not needed anymore?